joomla/joomla-cms
# [4.0] Replace atum/Service/HTML/Atum.php with CSS equivalent

### Summary of Changes
Currently, we use a PHP helper to set variants for admin color variables. This PR removes that requirement and uses CSS instead.


### Testing Instructions
Apply PR, compile SCSS and check administration looks the same. Ensure color selectors in admin template style settings still function.
